modules/new_profile/test/new_profileTest.php

```diff
@@ -79,36 +67,35 @@ function testNewProfilePageLoads()
      */
     function testNewProfileCreateCandidate(): void
     {
-        $this->markTestSkipped(
-            'Skipping tests until Travis and React get along better'
-        );
         $this->setUpConfigSetting("useEDC", "false");
```

Comment:
Should we also add test for the saturation that useEDC is set to true?

Reply:
This function only tests the "Create Candidate" function. 
If this test works fine with git action, I will add the useEDC in the future.